spec/features/posts/users_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'rails_helper'
+
+RSpec.describe UsersController, type: :request do
+  describe 'GET #index' do
+    it 'returns http success' do
+      get users_path
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe 'GET #show' do
+    it 'returns http success' do
+      get users_path(1)
+      expect(response).to have_http_status(:success)
+    end
+  end
+
+  describe 'GET #posts' do
+    it 'returns http success' do
+      get users_path(1)
+      expect(response).to have_http_status(:success)
+    end
+  end
+end
```

Comment:
- [ ] You should have the `posts_show_spec` file that tests:
* that asserts you can see the post's title.
* that asserts you can see who wrote the post.
* that asserts you can see how many comments it has.
* that asserts you can see how many likes it has.
* that asserts you can see the post body.
* that asserts you can see the username of each commentor.
* that asserts you can see the comment each commentor left.

- [ ]  Please kindly add all these tests to your file, those tests are for the `User` post show page 🙃